UnityProject/Assets/Scripts/Items/Implants/Organs/Vomit/LogicExtensions/StunOnVomit.cs

```diff
@@ -0,0 +1,15 @@
+﻿using HealthV2;
+using UnityEngine;
+
+namespace Items.Implants.Organs.Vomit.LogicExtensions
+{
+	public class StunOnVomit : MonoBehaviour, IVomitExtension
```

Comment:
it might be better to make this a reaction effect?

Reply:
nah, this is good because it means you can remove it or tweak it on specific species
also single responsibility principle